python_modules/dagster/dagster/_core/asset_graph_view/asset_graph_view.py

```diff
@@ -270,6 +270,57 @@ def compute_child_subset(
                 value=_ValidatedEntitySubsetValue(child_partitions_subset),
             )
 
+    def compute_mapped_subset(
+        self, to_key: T_EntityKey, from_subset: EntitySubset[AssetKey]
```

Comment:
Let's make from_subset `EntitySubset[EntityKey]`, nothing in here seems to be asset key specific